code/modules/mob/language.dm

```diff
@@ -463,7 +463,7 @@
 
 /datum/language/clown
 	name = "Clownish"
-	desc = "The language of clown planet. Mother tongue of clowns throughout the Galaxy."
+	desc = "The language of Clown University. Mother tongue of clowns throughout the Galaxy."
```

Comment:
```suggestion
	desc = "The language of Clown University. Mother tongue of clowns throughout the galaxy."
```
Small nitpick, while we are here. Galaxy in this case doesn't appear to be a proper noun, so it shouldn't be capitalized.